Documentation/platforms/aws.md

```diff
@@ -0,0 +1,77 @@
+
+* replication: etcd replicates all data to all members of the etcd cluster therefore more nodes and disks you have in your cluster and the stronger the isolation between failure domains, the smaller the probability that data persisted to an etcd cluster will be unavailable to an application
+* Point in time etcd snapshotting: the etcd v3 API introduced support for snapshotting clusters. The operation is cheap enough (completing in the order of minutes) to run quite frequently and the resulting archives can be archived in a storage service like Amazon Simple Storage Service (S3).
+* Choosing Amazon Elastic Block Storage (EBS) rather than EC2 Instance Store: an EBS volumes is a replicated network attached block device. As a replicated storage offering, EBS volumes have stronger storage safety guarantees than InstanceStore or Ephemeral volumes which have a life cycle associated with the life cycle of the attached EC2 instance. The life cycle of an EBS volume is not necessarily tied to an EC2 instance and can be detached and snapshotted independently which means that a single node etcd cluster backed by an EBS volume can provide a fairly reasonable level of data safety.
+
+### Performance/Throughput
+
+The performance of an etcd cluster is roughly quantifiable through latency and throughput metrics which are primarily affected by disk and network performance. Detailed performance planning information is provided in the [performance section](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/performance.md) of the etcd operations guide.
+
+#### Network
+
+AWS offers EC2 Placement Groups which allow the collocation of EC2 instances within a single Availability Zone which can be utilized in order to minimize network latency between etcd members in the cluster. It is important to remember that collocation of etcd nodes within a single AZ will have consequences for the overall availability of the etcd cluster. [Enhanced networking for EC2 instances](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/enhanced-networking.html) may also improve network performance of individual EC2 instances.
+
+#### Disk
+
+AWS provides two basic types of block storage: [EBS volumes](https://aws.amazon.com/ebs/) and [EC2 Instance Store](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html). As mentioned, an EBS volume is a network attached block device while instance storage is directly attached to the hypervisor of the EC2 host. EBS volumes will generally have higher latency, lower throughput and greater performance variance than Instance Store volumes. If performance, rather than data safety, is the primary concern it is highly recommended that instance storage on your EC2 instances be utilized. Remember that the amount of available instance storage varies by EC2 [instance types](https://aws.amazon.com/ec2/instance-types/) which may impose additional performance considerations.
```

Comment:
s/is the primary concern it is highly recommended that instance storage on your EC2 instances be utilized./is the primary concern, use instance storage on etcd's EC2 instances.